docs/integrations/overview.md

```diff
@@ -328,7 +328,7 @@ After that, the user receives a withdrawal voucher that they can use to withdraw
 
 ### Withdrawal
 
-Withdrawal of the ASSET implies sending the NFT to `withdrawal_manager` with the following message attached:
+Withdrawal of the ASSET implies sending the NFT to `withdrawal_voucher_contract` with the following message attached:
```

Comment:
these are different. `withdrawal_voucher_contract` - is NFT itself. `withdrawal_manager` -  exchanges NFT to the unbonded assets

Reply:
Well, in general when you sending NFT you interacting with NFT contract and not actually sending anything. In this particular case we call NFT contract method to transfer the ownership of this NFT item to `withdrawal_manager` and calling ReceiveNft handler in `withdrawal_manager` (which we may name as sending an NFT to `withdrawal_manager`)

Reply:
Up to you guys what you want to do with this. I was integrating with your protocol and followed the documentation. It states `withdrawal_mamager` in the docs so I looked up the address here https://docs.drop.money/deployments/mainnet. After looking though on chain transactions I worked out I actually needed to send it to the address listed as `withdawal_voucher_manager` in the contract addresses and the sendNFT message then worked.

Good luck!